test/env/logger.test.js

```diff
@@ -53,6 +53,12 @@ describe('#logger', function () {
     logger.debug('foo');
     logger.debug('foo', 'bar');
     logger.debug('%s %s', 'foo', 'bar', 'hop');
+    logger.debug(function(){return 'foo bar hop hop';});
+
+    // test logger.info()
+    logger.info('foo');
+    logger.info('foo', 'bar');
+    logger.info('%s %s', 'foo', 'bar', 'hop');
```

Comment:
Therefor I am testing it like that. See https://github.com/SassDoc/sassdoc/blob/add-test-utils-logger/test/env/logger.test.js#L95-L99